app/controllers/reports_controller.rb

```diff
@@ -0,0 +1,8 @@
+class ReportsController < ApplicationController
+  include ReportConcerns
+  before_action :set_from_and_to
+
+  def index(from = @from, to = @to)
+    @orders = Order.where(created_at: from..to)
```

Comment:
Create a scope for this. Pass values from param in that scope. Assign default values to scope in values not passed.

Reply:
problem